web/src/test/java/org/cbioportal/web/DataAccessTokenControllerTest.java

```diff
@@ -120,10 +124,6 @@ private void checkIfResponseMatches(ResponseEntity<DataAccessToken> response, Ht
         if (responseStatus != expectedStatus) {
             Assert.fail("Response from controller handler (" + responseStatus + ") did not match expected status : " + expectedStatus);
         }
-        MediaType responseType = response.getHeaders().getContentType();
```

Comment:
Why was this part of the test dropped? The response type should be JSON (unless the request fails due to authentication issues)